editor.cpp

```diff
@@ -99,13 +98,13 @@ void EditorWidget::onAsnDocumentUpdated(const QTextDocument &document)
     Q_UNUSED(document);
 
     QString filePath = textDocument()->filePath().toString();
-    ParsedDataStorage *storage = ParsedDataStorage::instance();
+    ParsedTree *tree = ParsedTree::instance();
```

Comment:
Maybe instead of name focusing on "parsing" let's try to find name explaining what it is. It just itches me ;)

maybe `ModelTree` ?